README.md

```diff
@@ -15,9 +15,12 @@ You don't need to install any other webserver or services.
 You need to copy the correct version from `WebGLTemplates/AirConsole` for Unity 2019 and below or `WebGLTemplates/AirConsole-2020` for Unity 2020+ to the WebGLTemplate folder you use.
 
 2. The content of the index.html / screen.html has changed.
-Search for `<script src="translation.js"></script>` and replace it with `<script src="airconsole-settings.js"></script>`, otherwise neither Translations nor Player Silencing will work.
+Open your controller's html file as well as the index.html in your WebGL template and search for `<script src="translation.js"></script>` and replace it with `<script src="airconsole-settings.js"></script>`, otherwise neither Translations nor Player Silencing will work.
 
-3. Ensure that obsolete API devices, device_id and server_time_offset are updated:
+3. The version of the airconsole api has changed.
+Search for a script usage like `<script src="https://www.airconsole.com/api/airconsole-1.8.0.js"></script>` and replace it with `<script src="https://www.airconsole.com/api/airconsole-1.9.0.js"></script>`
```

Comment:
Same here, wouldn't it be better to tell them to always use the latest api version?